recipe/meta.yaml

```diff
@@ -578,6 +579,7 @@ outputs:
         - {{ pin_subpackage('libopenvino-auto-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-hetero-plugin', exact=True) }}
         - {{ pin_subpackage('libopenvino-auto-batch-plugin', exact=True) }}
+        - tbb-devel =2021.5  # to property find libs via openvino.pc
```

Comment:
its very strange that you are adding the `=2021.5` pinnings.

can you unpin things and allow the conda-forge global pinnings to take effect?
Is that appropriate for this package?

Reply:
`tbb-devel` provides `run_exports` as https://github.com/conda-forge/tbb-feedstock/blob/main/recipe/meta.yaml#L106
So, any TBB 2021.5 and newer can be used.

I've removed this pinning to avoid confusion.